metal-deployment/base/Dockerfile

```diff
@@ -1,49 +1,39 @@
-FROM python:3.8-buster
+FROM debian:10
 
-ENV VERSION_ANSIBLE=2.9.13 \
+ENV VERSION_ANSIBLE=2.9.15 \
     VERSION_CT=0.9.0 \
-    VERSION_HELM=3.4.0 \
-    VERSION_STERN=1.11.0 \
-    VERSION_VAGRANT=2.2.9
-
-# vagrant is required for running the vagrant dynamic inventory script from within the container...
-ARG VAGRANT_PACKAGE_URL=https://releases.hashicorp.com/vagrant/${VERSION_VAGRANT}/vagrant_${VERSION_VAGRANT}_x86_64.deb
+    VERSION_HELM=3.4.0
 
 RUN set -x \
- && export CLOUD_SDK_REPO="cloud-sdk-buster" \
- && echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list \
- && curl -f https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - \
  && apt-get update \
  && apt-get install --yes --no-install-recommends \
         apt-transport-https \
         ca-certificates \
         curl \
         gnupg2 \
-        google-cloud-sdk \
         software-properties-common \
-        connect-proxy \
-        libvirt-dev \
-        ruby-dev \
-        rsync \
-        yamllint \
+ && curl -f https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - \
  && curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
+ && add-apt-repository "deb [arch=amd64] http://packages.cloud.google.com/apt cloud-sdk-buster main" \
  && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian buster stable" \
  && apt-get update \
- && apt-get install --yes --no-install-recommends docker-ce  \
+ && apt-get install --yes --no-install-recommends \
+        google-cloud-sdk \
+        docker-ce  \
+        connect-proxy \
+        python3 \
+        python3-pip \
+        python3-setuptools \
+        python3-wheel \
+        rsync \
+        yamllint \
  && curl -fsSL https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash -s -- --version "v${VERSION_HELM}" \
- && pip install --upgrade pip \
- && pip install ansible==${VERSION_ANSIBLE} Jinja2==2.11.1 netaddr==0.7.19 humanfriendly==8.1 openshift==0.11.2 paramiko==2.7.1 \
- && curl -fo vagrant.deb $VAGRANT_PACKAGE_URL \
- && dpkg -i vagrant.deb \
- && rm -f vagrant.deb \
- && vagrant plugin install vagrant-libvirt \
- && curl -Lo stern https://github.com/wercker/stern/releases/download/${VERSION_STERN}/stern_linux_amd64 \
- && chmod +x stern \
- && mv stern /usr/local/bin/ \
+ && pip3 install ansible==${VERSION_ANSIBLE} Jinja2==2.11.1 netaddr==0.7.19 humanfriendly==8.1 openshift==0.11.2 paramiko==2.7.1 \
```

Comment:
We often execute `pip` in CI jobs to install something specific for testing. `pip` has to be available.

Reply:
Good ideas, done